src/FluentValidation.DependencyInjectionExtensions/FluentValidation.DependencyInjectionExtensions.csproj

```diff
@@ -31,8 +31,8 @@
     <ItemGroup>
         <Compile Include="..\CommonAssemblyInfo.cs" Link="CommonAssemblyInfo.cs" />
     </ItemGroup>
-    <ItemGroup>
-        <PackageReference Include="Microsoft.AspNetCore.Mvc" Version="2.1.0" />
+    <ItemGroup>
+      <PackageReference Include="microsoft.extensions.dependencyinjection.abstractions" Version="2.2.0" />

```

Comment:
This is the minimum dependency for this to work. I suggest you move anything AspNetCore related to another assembly so that this stays net framework compatible

Reply:
Please can this be downgraded to 2.1.0? FluentValidation supports .NET Core 2.1 as the lowest supported version, as this is the LTS release (2.2 is not LTS).